config-example.yaml

```diff
@@ -45,8 +50,10 @@ ca: #ACME and none ACME CA
       url: https://acme-v02.api.letsencrypt.org/directory
       roots: https://letsencrypt.org/certificates/
       description: "Let's Encrypt Official"
-      logopath: "https://foo.bar/logo.png"
+      logopath: "/images/logo.png"
       checkNameservers:
+       # These nameservers are used to check if and when the DNS entries are propagated
+       # so the ACME provider can read the challenges
```

Comment:
CA is doing ACME validation. Why we need to read the challenge?

Reply:
Because it is unpredictable when the challenge will become active on the DNS servers. This is especially valid for OTC.
Uses a built-in function in lego.

Reply:
Yes, its unpredictable. That's why ACME does polling. What lego is doing here? And why?